docs/modules/clusters/pages/ucn-dynamic-client.adoc

```diff
@@ -1,13 +1,65 @@
 = Configure Client
-:description: Clients use the dynamic configuration API to update {ucn} on the cluster. 
+:description: Clients use the dynamic configuration API to create new, and update existing, {ucn} on the cluster.
 :page-enterprise: true
 :page-beta: false
 
 {description}
 
-You must create a _Client.java_ file containing the configuration and save it to your configuration folder. 
+== Dynamically creating new {ucn}
 
-NOTE: Ensure that you have configured your data structure with the same {ucn} referenced in your client configuration.
+You must create a _Client.java_ file containing the required code and save it to your configuration folder.
+
+NOTE: {ucn} must be statically enabled on the member side before clients are allowed to dynamically create {ucn}.
+
+The file will be similar to the following:
```

Comment:
```suggestion
The created file will be similar to the following:
```

Reply:
I'm fine with this change, but just an FYI that it's used in several other UCN documentation pages as well (it was already on this page).